### ISSUE TITLE
Does the framework intend to continue to be updated?

### ISSUE THREAD

**Issue by ccworld1000** [dmsurti/AssimpKit#121]
Does the framework intend to continue to be updated?

**Comment by dmsurti** [comment]
@ccworld1000 Yes I am working on a newer version but to be honest with very little time at disposal due to other work commitments, I can’t commit to an ETA. 

Any specific features you can mention would be useful. Thanks

**Comment by ccworld1000** [comment]
Does it support FBX < < = > > obj conversion?
Mutual conversion

**Comment by ccworld1000** [comment]
Whether the library file can be cropped can be customized and imported in pod as needed

**Comment by smhk** [comment]
@dmsurti ,

DM, everyone surely **thanks you greatly** for your work. Thanks!

But nobody is going to use carthage anymore. (Every single client we have, corporations, has eliminated carthage.)

Surely, after all your amazing work, SPM is needed?

Very best, JP

**Comment by dmsurti** [comment]
@smhk Thanks for the appreciation 👍

Despite the best of intentions, I am unable to find time to make the necessary updates due to other conflicting priorities. However, if you can submit a PR with the necessary changes, I will be more than happy to review and merge it. 

Thanks.
